src/dash-adapter.js

```diff
@@ -486,9 +487,35 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
   /**
    * apply ABR restrictions
    * @private
+   * @param {Object} ABRConfig - abr config
    * @returns {void}
    */
-  _maybeApplyAbrRestrictions(): void {
+  _maybeApplyAbrRestrictionsByBitrate(ABRConfig: Object): void {
+    if (ABRConfig.restrictions) {
+      const restrictions = ABRConfig.restrictions;
+      const minBitrate = restrictions.minBitrate ? restrictions.minBitrate : 0;
+      const maxBitrate = restrictions.maxBitrate ? restrictions.maxBitrate : Infinity;
+      if (maxBitrate < minBitrate) {
+        if (restrictions.minBitrate >= 0) {
+          this._config.capLevelToPlayerSize = false;
```

Comment:
this is by default false.
If an app set this and wants to apply other means of capping it needs to set it to false by itself.